core/tests/test_utils.py

```diff
@@ -2260,6 +2260,30 @@ def _create_valid_question_data(self, default_dest_state_name):
         state.interaction.default_outcome.dest = None
         return state
 
+
+GenericTestBase = AppEngineTestBase
+
+
+class LinterTestBase(GenericTestBase):
```

Comment:
Any specific reason for this to be a base from **AppEngine**TestBase ?

Reply:
Yes because that class is inheriting from `unittest.TestCase` and also has `swap()` method and many other methods like checking for the use of `assertRaisesRegexp()`